src/sci/nrepl/browser_server.clj

```diff
@@ -189,16 +182,3 @@
          websocket-port 1340}}]
   (start-nrepl-server! {:port nrepl-port})
   (start-websocket-server! {:port websocket-port}))
-
-(defmacro export [& var-names]
-  (let [var-names (set var-names)]
-    (doseq [[k v] (ns-publics *ns*)]
-      (when-not (contains? var-names k)
-        (alter-meta! v assoc :private true)))))
-
-(export
```

Comment:
I still prefer that we make stuff explicitly public or private so I can still make breaking changes in the private functions